ansible/www-standalone/resources/config/nodejs.org

```diff
@@ -74,54 +95,98 @@ server {
     access_log /var/log/nginx/nodejs/nodejs.org-access.log nodejs;
     error_log /var/log/nginx/nodejs/nodejs.org-error.log;
 
+    # We gzip the responses that NGINX does for the following mime-types. We don't use gzip_static as otherwise it would
+    # request a pre-gzip version of each file on the file system, which would hammer the file system with requqests.
     gzip on;
     gzip_disable "MSIE [1-6]\.";
     gzip_types text/plain text/css application/javascript text/xml application/xml application/xml+rss image/svg+xml;
 
+    # These directives are used to prevent our server to continuously attempt to open the same requested files on the file_system
+    # W use this so that file_system accesses get cached during this period of time.
+    # For specific Location directives, such as /dist we also cache the 404's and any other Server Error
     open_file_cache max=100000 inactive=300s;
     open_file_cache_valid 120s;
     open_file_cache_min_uses 2;
     open_file_cache_errors off;
 
+    # We set a default language to "en". This is used for the Localized 404 pages
     set $lang en;
 
     root /home/www/nodejs;
     default_type text/plain;
     index index.html;
+    
+    # We set a default cache of 1 hour and a max age of 4 hours
     add_header Cache-Control "public, max-age=3600, s-maxage=14400";
 
+    # Sets a custom 404 error page for the whole Server region
+    # we use the Next.js generated 404 page for all the 404's of our Website
+    # including for binaries, assets and et cetera.
     error_page 404 @localized_404;
 
+    # As we don't have a /index.html, we redirect to /en
     location = / {
-        rewrite ^ /en break;
+        rewrite ^ /en permanent;
```

Comment:
This rewrite is done with code 302 (temporary) at the CF level

Reply:
I don't know.

Reply:
I mean, we should probably remove that rewrite from there, as CF would cache this one eitherways.